manifests/0000_90_cluster-authentication-operator_01_prometheusrbac.yaml

```diff
@@ -38,6 +38,24 @@ rules:
   - list
   - watch
 ---
+# oauth-apiserver role
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: prometheus-k8s
+  namespace: openshift-oauth-apiserver
```

Comment:
Is this annotation intentionally ignored from all the resources?

`include.release.openshift.io/self-managed-high-availability: "true"`

As I understand it, CVO would not create the resource if the annotation was missing in default cluster profile mode. Please correct me if I am wrong.

Reply:
I think that should still be in. Good catch!